paper/paper.md

```diff
@@ -0,0 +1,170 @@
+mathematical models used in the sport.
+
+It is used in multiple applications including the development of websites
+and apps for archers, conducting performance analysis for elite archers and governing
+bodies, and in ongoing sports science research.
+
+
+# Statement of Need
+
+In the sport of archery athletes shoot targets at a variety of distances.
+Further, these targets can be of different sizes and have different scoring systems
+associated with them.
+The overall goal is always the same, however; to shoot arrows as accurately and
+precisely as possible.
+For the purposes of analysis, it is therefore useful to be able to extract some comparative
+measure of performance across all targets and distances.
+As a precision-based sport, archery lends itself well to mathematical
+modelling with examples in literature being @lane2013construction, @park2014modelling, and the
+2023 handicap system by Archery GB [@handicaps].
+Like many scientific models, these are described in papers where they are used by
+the authors, but no software implementation is publicly available.
+This presents a barrier to further research and development for those who understand
+how to use the models, but not how they are constructed or implemented.
+_archeryutils_ has been developed as a tool to remove this barrier and improve
+reproducibility.
+
+Archery score data is increasingly
+readily available in electronic formats, be it from competition results
+(e.g. on [ianseo](https://ianseo.net/), [tamlynscore](https://www.tamlynscore.co.uk/), etc.)
+or the various scoring applications used by archers of all levels.
+The _archeryutils_ API is easily combined with Python
+data-processing packages, for example
+pandas [@The_pandas_development_team_pandas-dev_pandas_Pandas; @mckinney2011pandas] or polars [@polars],
+enabling users to perform analysis on these datasets
+without needing to reimplement models themselves.
+
+
+# Software Description
+
+_archeryutils_ is a Python package distributed through the
+[Python Package Index (PyPI)](https://pypi.org/project/archeryutils/) with full documentation,
+including the API, provided online at
+[archeryutils.readthedocs.io](https://archeryutils.readthedocs.io/)
+
+At the core of _archeryutils_ are classes representing the core components of
+the sport.
+Perhaps the most notable of these are the `Target`, providing a representation
+of a target of a particular size, distance, and face type, and the
+`Round`, which builds upon this to represent of a series of targets shot at with a
+number of arrows.
+As well as a constructors for creating generic targets and rounds, _archeryutils_ comes
+with many popular national and international rounds predefined for users.
+These components provide a starting point for a variety of applications.
+
+Implementation of mathematical models for precision is provided by the `handicaps`
+module.
+Available models include both Archery GB handicap schemes of @lane2013construction
```

Comment:
```suggestion
Available models include both the Archery GB handicap schemes of @lane2013construction
```

Reply:
Pushing word limit so will leave this for now.